### ISSUE TITLE
Missing Wiki Page for SC2011

### ISSUE THREAD

**Issue by aphedges** [koalaman/shellcheck#1833]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2011
- My shellcheck version (`shellcheck --version` or "online"): online
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/env bash
ls | xargs wc -w
```

#### Here's what shellcheck currently says:
```sh
Line 2:
ls | xargs wc -w
^-- SC2011: Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.
```

The check name links to https://github.com/koalaman/shellcheck/wiki/SC2011, which is empty.

#### Here's what I wanted or expected to see:
An explanation for this check on the wiki.

**Comment by dimo414** [comment]
Funny, looks like this dates all the way back to [2013](https://github.com/koalaman/shellcheck/commit/e5e08df1d980723e4f24d3b6667706633902bb28) and hasn't been created. I took a swing at adding something, feel free to expand on it!

I went with your example here, but it would probably be good to identify a better one. Unless I'm mistaken `ls | xargs wc -w` could simply be `wc -w *`.

**Comment by aphedges** [comment]
The command I gave was asimplified version I was able to create that still triggered the check. My actual code was `ls example/* | xargs -I % wc -w %`, which has different output from `wc -w example/*`. The latter example has extra formatting and a row for the total, which I was trying to avoid.

I am pretty new to Bash/Coreutils/xargs/etc., so a more experienced person can probably come up with a better example. Regardless, what you wrote on the wiki is much better than nothing. Thanks!

**Comment by dimo414** [comment]
Ah, using `-I %` causes `xargs` to invoke `wc` separately for each file, so the final `total` line isn't printed by `wc`. Fair enough :) `xargs -n1` would also work, FWIW, but I'm just nitpicking. Welcome to the wonderful world of Bash!

**Comment by aphedges** [comment]
I wasn't aware the `-n` flag would also work. I'm still learning how to use `xargs` well. Thanks for the tip!

Do you think what's now on the wiki page is enough to close this ticket?

**Comment by dimo414** [comment]
`xargs` is pretty magic, but it's a great tool to have in the toolbelt. Feel free to close this if you feel the current content is good enough for now (I'm not able to close this issue).

**Comment by aphedges** [comment]
It's definitely isn't a missing page anymore, so the problem seems solved. I'm closing this.
